eZ/Publish/Core/Search/Legacy/Content/IndexerGateway.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Search\Legacy\Content;
+
+use DateTime;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\ResultStatement;
+use Doctrine\DBAL\ParameterType;
+use eZ\Publish\SPI\Persistence\Content\ContentInfo;
+use eZ\Publish\SPI\Search\Content\IndexerGateway as SPIIndexerGateway;
+use Generator;
+
+/**
+ * @internal
+ */
+final class IndexerGateway implements SPIIndexerGateway
+{
+    /** @var \Doctrine\DBAL\Connection */
+    private $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function getStatementContentSince(DateTime $since, bool $count = false): ResultStatement
```

Comment:
```suggestion
    public function getStatementContentSince(DateTimeInterface $since, bool $count = false): ResultStatement
```

Please use DateTimeInterface instead of DateTime

Reply:
Fixed f165feabd44271c587c553edc01c571796277f93